test/test_sparse.py

```diff
@@ -3978,10 +3992,13 @@ def test_basic(self):
         self.assertEqual(r.dense_dim(), 1)
         self.assertEqual(r._dimV(), 1)
         self.assertEqual(r._nnz(), 0)
```

Comment:
This test case required some minor changes, but the TODO anticiapted them, so I think it is fine.